plexapi/myplex.py

```diff
@@ -936,6 +936,27 @@ def userState(self, item):
         data = self.query(f"{self.METADATA}/library/metadata/{ratingKey}/userState")
         return self.findItem(data, cls=UserState)
 
+    def isPlayed(self, item):
+        """ Returns True if this item is played. """
```

Comment:
Add documentation for the `item` parameter.

Explicitly state that `item` must be a video object from `searchDiscover()`.

Same for the other two methods.

Reply:
item can be also video object if it's from new plex agent. i.e media object with guid `plex://<hash>`

Reply:
Right, I forgot about that. Then just the same as the `item` parameter from `userState()` above.

But update the doc sting to say "Return True if the item is played on Discover." (same for the other two methods) so it doesn't get confused with the mixin methods.

Reply:
Added: e986b56

agents doc:
- https://support.plex.tv/articles/200241558-agents/

Reply:
Added discover note: c61c733